src/Event/ApiGateway/ApiGatewayHandler.php

```diff
@@ -0,0 +1,20 @@
+<?php declare(strict_types=1);
+
+namespace Bref\Event\ApiGateway;
+
+use Bref\Context\Context;
+use Bref\Event\Handler;
+
+/**
+ * Handles ApiGateway events.
+ */
+abstract class ApiGatewayHandler implements Handler
```

Comment:
Could be `WebsocketHandler` instead? (to be more precise, and we already are in the `ApiGateway` namespace so no risk of collision in the future.

Reply:
Sure. I see the pattern. Just looked over the other event handlers and they're doing the same thing. I'll align it.